### ISSUE TITLE
Make Warp Sync the default Sync mode

### ISSUE THREAD

**Issue by bkchr** [paritytech/substrate#12250]
We should make warp sync the default sync mode. Only if someone starts with `--state-pruning archive` we should switch to the catch up sync mode, aka download and execute every block.

This depends on: https://github.com/paritytech/substrate/issues/12249

**Comment by nazar-pc** [comment]
I am a bit concerned about this because it implies a bunch of assumptions about properties of the chain like consensus and with Substrate being a generic framework it would be unfortunate to have our chain broken because the default sync is no longer compatible with it.

I'm not sure what is the scope of this issue though, maybe we can patch it and override the default mechanism somehow even after this change :shrug:

**Comment by bkchr** [comment]
Good point and clearly something that should be considered while doing this :)

**Comment by Polkadot-Forum** [comment]
This issue has been mentioned on **Polkadot Forum**. There might be relevant details there:

https://forum.polkadot.network/t/community-bootnodes-and-snapshot-providers/630/1
